capstone/src/main/java/com/google/sps/AssociationAnalysis.java

```diff
@@ -7,14 +7,26 @@
 /** Calculates association scores from a list of sentiments per entity mention */
 public class AssociationAnalysis {
 
+  HashMap<String, AssociationResult> res;
+
+  public AssociationAnalysis() {
+    res = new HashMap<String, AssociationResult>();
+  }
+
+  public AssociationAnalysis(ArrayList<AssociationResult> init) {
```

Comment:
Should we use an argument more descriptive than `init` that might help callers that are less familiar with this code?